cheroot/server.py

```diff
@@ -696,7 +696,6 @@ def parse_request(self):
         else:
             if not success:
                 return
-
```

Comment:
This is unrelated.

Reply:
Addressed in 1be10238164f82a7d83bfa8a404a3f1f9961cdf3.